niceman/resource/ssh.py

```diff
@@ -26,7 +28,7 @@ class SSH(Resource):
     name = attr.ib()
 
     # Configurable options for each "instance"
-    host = attrib(doc="DNS or IP address of server")
+    host = attrib(default='127.0.0.1', doc="DNS or IP address of server")
```

Comment:
why there should be a default address??

Reply:
DNS is generally associated with "server". Here better just to use "Name or IP address"